### PR TITLE
Fixed background of profile picture

### PR DESCRIPTION
**Pull request by Cililocwee**
## Type of change

[] New feature
<br />
[x] Bug fix
<br />
[] Documentation
<br />
[] Code refactor/tech debt
<br />
[] Infrastructure

## Description
The profile picture needed to be circular instead of an oval.

## Ticket link

[Ticket CDW-51](https://linear.app/cdw/issue/CDW-51/profile-icon-background-is-not-a-circle)

## Part of a back end implementation?
[x] No
<br />
[] [Yes]()  

## How to test

1. Checkout to this branch
2. From /frontend, npm start
3. Log into the app
4. Check that the picture is a circle

## Screenshots

image
image

### CODE REVIEW

**Review by noah-milliken — COMMENTED** [review]
Thats a fine looking profile picture, Corrie.

**Comment by jtabba** [comment]
Why's this a 19 file change?

You may need to `git fetch origin; git rebase origin prod` but investigate why first (commit history is whats making me think this could be a solution)

File: frontend/src/pages/home.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    },
+    {
+      headline: "Lorem Ipsum 3",
+      content:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      linkLabel: "Lorem Ipsum 3 >",
+      url: "/loremipsum3",
+    },
+  ];
+
```
Comment (jtabba) [inline frontend/src/pages/home.tsx:48]: Danny has moved these into their own `constants` file. Import them from there instead of re declaring them

**Review by jtabba — CHANGES_REQUESTED** [review]

File: frontend/src/pages/home.tsx (hunk 1 above)
Comment (Cililocwee) [reply frontend/src/pages/home.tsx:48]: This was an error. For some reason my push had retained an old home.tsx that wasn't in it's own directory. I've removed it to match prod. Everything should be fine now

**Comment by Cililocwee** [comment]
Okay, @jtabba, the extraneous home.tsx has been removed, so let me know if there's anything else to change on this branch!

**Review by jtabba — APPROVED** [review]

**Pull request merged**